request.go

```diff
@@ -465,3 +489,15 @@ func (r *Request) fmtBodyString() (body string) {
 
 	return
 }
+
+func (r *Request) selectAddr(addrs []*net.SRV, path string, attempt int) string {
+	if addrs == nil {
```

Comment:
Maybe I didn't understand, but I think [this][1] test case already cover this situation. 

[1]: https://github.com/go-resty/resty/pull/51/files/9a10560ae3e2c992116bf965323d403082b7ad5c#diff-283b9a8bcf7d3d14aaaa296a2bfd884cR1428